platforms/quorum/configuration/roles/create/crypto/ibft/tasks/nested_main.yaml

```diff
@@ -0,0 +1,35 @@
+# This task checks if the crypto material is already stored in the vault or not
+- name: Check if nodekey already present in the vault
+  shell: |
+    vault read -field=nodekey secret/{{ component_ns }}/crypto/{{ peer.name }}/quorum
+  environment:
+    VAULT_ADDR: "{{ vault.url }}"
+    VAULT_TOKEN: "{{ vault.root_token }}"
+  register: vault_nodekey_result
+  ignore_errors: yes
+
+# This task creates the build directory if it does not exist
+- name: Create build directory if it does not exist
+  file:
+    path: build/{{ component_name }}/{{ peer.name }}
+    state: directory
+    mode: '0755'
+    recurse: yes
+  when: vault_nodekey_result.failed == True
+
+- name: Generate enode url for each node and create a geth account and keystore
+  shell: |
+    bootnode --nodekey=build/{{ component_name }}/{{ peer.name }}/nodekey --writeaddress > build/{{ component_name }}/{{ peer.name }}/enode
```

Comment:
add bootnode and geth roles in the meta, so that the dependencies are strong. That way we do not have to explicitly call bootnode and geth roles